discord/cog.py

```diff
@@ -740,6 +742,34 @@ def _resolve_name(self, name: str, package: Optional[str]) -> str:
         except ImportError as e:
             raise errors.ExtensionNotFound(name) from e
 
+    def load_extension_from_path(
```

Comment:
This could be `load_extensions_from_path` to reflect loading multiple extensions

Reply:
Okay i will fix it

Reply:
Fixed